adafruit_ltr390.py

```diff
@@ -304,16 +304,16 @@ def initialize(self):
         # ltr.configInterrupt(true, LTR390_MODE_UVS);
 
     def _reset(self):
-        try:
-            self._reset_bit = True
-        except OSError:
-            # The write to the reset bit will fail because it seems to not ACK before it resets
-            pass
-
-        sleep(0.1)
-        # check that reset is complete w/ the bit unset
-        if self._reset_bit:
-            raise RuntimeError("Unable to reset sensor")
+        # The LTR390 software reset is ill behaved and can leave I2C bus in bad state.
+        # Instead, just manually set register reset values per datasheet.
+        with self.i2c_device as i2c:
+            i2c.write(bytes([_CTRL, 0x00]))
```

Comment:
can ya make these () tuples instead of [] lists

Reply:
yep